### PR TITLE
CSHARP-1849: Run Plain Auth Tests

### PR DESCRIPTION
**Pull request by durran**

### CODE REVIEW

**Comment by DmitryLukyanov** [comment]
can you post an EG link?

File: build.cake [hunk 1]
```diff
@@ -180,6 +180,24 @@ Task("TestAwsAuthentication")
             );
         });
 
+Task("TestPlainAuthentication")
+    .IsDependentOn("Build")
+    .DoesForEach(
```
Comment (rstam) [inline build.cake:185]: There's only one matching `.csproj` file. Is there a simpler way to do this that doesn't involve looping over a single value?

**Review by rstam — APPROVED** [review]
Another one that is a branch on a branch...

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request closed without merging**